src/static/facade/rag_poisoning_template.css

```
@@ -0,0 +1,187 @@
+
+.llmforge-facade-output {
+  margin: 0;
+  padding: 12px;
+  border-radius: 6px;
+  overflow: auto;
+  max-height: 340px;
+  background: #f6f8fa;
+  border: 1px solid #dde3ea;
+  white-space: pre-wrap;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)